web/js/landingView.js

```diff
@@ -75,10 +75,12 @@
     Array.prototype.map.call(fields, function (field) {
       field.value = '';
       field.checked = false;
-      user.focus();
-      user.addEventListener('keyup', onKeyup);
-      room.addEventListener('focus', onFocus);
-      user.addEventListener('focus', onFocus);
+      if (user) {
+        user.focus();
+        user.addEventListener('keyup', onKeyup);
+        user.addEventListener('focus', onFocus);
+      }
+      if (room) room.addEventListener('focus', onFocus);
```

Comment:
Are these changes related to the Adobe tracking code? Are they improvements for the landing page? I do not see this code loading in the /thanks page, and I don't see it executing on the landing page. Maybe remove these changes?

Reply:
I am seeing null pointer exception while loading the landing page after merging adobe tracking code changes
Error:
landingView.js:78 Uncaught (in promise) TypeError: Cannot read property 'focus' of null
    at landingView.js:78

Reply:
I was concerned about these event handlers not being set. But it looks like we can remove the `resetForm()`, `onKeyup()`, and `onFocus()` functions entirely (along with references to `room`). They are legacy from the old design, where there were multiple input fields.

For now, let's just leave this fix in. We can consider cleaning this stuff up later.